content/en/real_user_monitoring/guide/setup-feature-flag-data-collection.md

```diff
@@ -192,6 +192,120 @@ For more information about initializing Amplitude's SDK, see Amplitude's [Androi
 Amplitude does not support this integration. Create a ticket with Amplitude to request this feature.
 
 
+{{% /tab %}}
+{{< /tabs >}}
+
+### ConfigCat integration
+
+{{< tabs >}}
+{{% tab "Browser" %}}
+
+Initialize the ConfigCat Javascript SDK by subscribing to the `flagEvaluated` event that reports feature flag evaluations to Datadog:
```

Comment:
**💬 suggestion:** ‏The phrasing looks weird to me. When reading this, I would understand that the "flagEvaluated" event has logic to report feature flag evaluations to Datadog out of the box. I don't think it describes the following code snippet correctly.


```suggestion
When initializing the ConfigCat Javascript SDK, subscribe to the `flagEvaluated` event and report feature flag evaluations to Datadog:
```